src/components/Directions.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <ol v-if="this.directions.length">
+    <li :key='i' v-for="(direction, i) in directions">
+      {{direction}}  
+    </li>
+  </ol>
+</template>
+
+<script>
+export default {
+  props: ["park"],
+  data() {
+    return {
+      directions: []
+    }
+  },
+  mounted() {
```

Comment:
Thanks for adding these annotations.  It definitely makes things easier when we want to recall what we've already discussed, and where code should exist.